org.aliflang.lang.yml

```diff
@@ -0,0 +1,149 @@
+app-id: org.aliflang.lang
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
```

Comment:
Any reason for not bumping this to 20.08?

Reply:
yes, I can't build webkitgtk-2.4.11 with 20.08 but build successful with 19.08
I'll rebuild it and send you the error

Reply:
```
/usr/include/unicode/unorm.h:218:1: note: declared here
  218 | unorm_normalize(const UChar *source, int32_t sourceLength,
      | ^~~~~~~~~~~~~~~
  CXX      Source/WebCore/platform/image-decoders/webp/libPlatform_la-WEBPImageDecoder.lo
Source/WebCore/platform/graphics/SegmentedFontData.cpp: In member function ‘virtual bool WebCore::SegmentedFontData::containsCharacters(const UChar*, int) const’:
Source/WebCore/platform/graphics/SegmentedFontData.cpp:65:9: error: expected ‘;’ before ‘if’
   65 |         if (!containsCharacter(c))
      |         ^~
make[1]: *** [GNUmakefile:51072: Source/WebCore/platform/graphics/libPlatform_la-SegmentedFontData.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/run/build/webkitgtk'
make: *** [GNUmakefile:25837: all] Error 2
Error: module webkitgtk: Child process exited with code 2

```